src/App.js

```diff
@@ -1,11 +1,17 @@
 import logo from './logo.svg';
 import './App.css';
-
+import SearchForm from './SearchForm/SearchForm';
+import Counter from './Counter/Counter';
+import GenreSelect from './GenreSelect/GenreSelect';
+import "bootstrap/dist/css/bootstrap.min.css";
 
 function App() {
+  const geners=['Action', 'Comedy', 'Drama', 'Fantasy'];
```

Comment:
mock data should not be a part of ui component, let's put it in a separate file located near GanreSelect: 
<img width="182" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/b33b6db7-e57d-4999-ae87-900b1bfa4491">

after you change a folder structure

Reply:
fixed